gr-iio/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 ########################################################################
 include(GrBoost)
 
-find_package(AD9361 QUIET)
+find_package(AD9361)
```

Comment:
Do we need both this change and the one below?

Reply:
Well, not necessarily. But nowhere else in the CMakeFiles find_package is used with the quiet option.
I personally prefer more verbosity when developing.

As gr-iio will be build independently whether AD9361 is found or not, the user should get a hint about the consequences.

Reply:
Just detected

find_package(Qt5Widgets QUIET)